### PR TITLE
#35 Final review and polishing

### PR DESCRIPTION
**Pull request by enriquemeseguerinbibo**
From issue #35.

Addressed final open comments of documentation PRs and polished documentation overall.

### CODE REVIEW

File: docs/documentation/maya/tools.md [hunk 1]
```diff
@@ -126,6 +126,11 @@ In the specific case of AdnMuscle deformers, the AdonisFX Paint Tool will displa
     4. When selecting a segment in the list the two scene objects that form the root and tip of the segment will get selected as well, facilitating its identification.
     5. If more than one segment was added to the system, then the paint tool will normalize the weights automatically after a stroke has been completed, meaning that the addition of all slide on segment constraint weights in a vertex will always add up to a maximum value of 1.0.
 
+    <figure>
+      <img src="images/paint_tool_sos_attribute.png"> 
+      <figcaption><b>Figure 7</b>: AdonisFX Paint Tool listing multiple segments.</figcaption>
```
Comment (josevalverdeinbibo) [inline docs/documentation/maya/tools.md:131]:
```suggestion
      <figcaption><b>Figure 8</b>: AdonisFX Paint Tool listing multiple segments.</figcaption>
```
Please check the number of the next figures.

**Review by josevalverdeinbibo — DISMISSED** [review]
Great job addressing the pending questions! Just some comments to check.

File: docs/documentation/maya/ribbon.md [hunk 2]
```diff
@@ -55,73 +55,73 @@ In order to provide more artistic control, some key parameters of the AdnRibbonM
...
 
 #### Solver Attributes
- - **Enable** (Boolean, True): Flag to enable or disable the deformer computation.
- - **Iterations** (Integer, 10): Number of iterations that the solver will execute per simulation step. Greater values mean greater computational cost.
+ - **Enable** (Boolean, True, Keyable): Flag to enable or disable the deformer computation.
+ - **Iterations** (Integer, 10, Keyable): Number of iterations that the solver will execute per simulation step. Greater values mean greater computational cost.
      - Has a range of \[1, 10\]. Upper limit is soft, higher values can be used.
  - **Material** (Enumerator, "Muscle"): Solver stiffness presets per material. The materials are listed from lowest to highest stiffness. There are 7 different presets:
     <ul><li>Fat: 10^7^</li><li>Muscle: 5e^3^</li><li>Rubber: 10^6^</li><li>Tendon: 5e^7^</li><li>Leather: 10^8^</li><li>Wood: 6e^9^</li><li>Concrete: 2.5e^10^</li></ul>
  - **Stiffness Multiplier** (Float, 1.0): Multiplier factor to scale up or down the material stiffness.
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/ribbon.md:63]: Done.

File: docs/documentation/maya/edge_evaluator.md [hunk 3]
```diff
@@ -12,6 +12,11 @@ The Edge Evaluator Node requires the following inputs to be provided:
 > [!NOTE]
 > - All input geometries must have the same number of vertices and edges.
 
+<figure markdown>
+  ![Edge Evaluator node](images/edge_evaluator.png)
+  <figcaption><b>Figure 1</b>: Edge Evaluator Attribute Editor</figcaption>
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/edge_evaluator.md:17]: Done.

File: docs/documentation/maya/edge_evaluator.md [hunk 4]
```diff
@@ -42,3 +47,8 @@ The evaluator node can be used to drive the [activations of an AdnSimshape](sims
 2. Go to the AdonisFX Menu > Edit Simshape > *Connect Activations Plug*.
 
 In order to disconnect the plug, repeat the selection and instead of pressing *Connect Activations Plug* press *Disconnect Activations Plug*.
+
+<figure markdown>
+  ![Edge Evaluator example](images/edge_evaluator_example.png)
+  <figcaption><b>Figure 1</b>: AdnSimshape deformer with activations provided by the AdnEdgeEvaluator node (activations debugger enabled)</figcaption>
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/edge_evaluator.md:53]: Done.

File: docs/documentation/maya/simshape.md [hunk 5]
```diff
@@ -215,7 +215,10 @@ The features that can be visualized with the debugger in the AdnSimshape deforme
 
 Enabling the debugger and selecting one of these constraints will draw lines from the influenced vertices in the simulated mesh to their corresponding reference vertices. 
 
-<!-- We might want to add a screenshot here of a debug functionality like fibers -->
+<figure markdown>
+![skin editor debug menu](images/simshape_debug.png)
+<figcaption><b>Figure 4:</b> AdnSimshape Muscle Fibers and Collision Constraints debugging</figcaption>
```
[the remark this answers is not in the document]
Comment (josevalverdeinbibo) [reply docs/documentation/maya/simshape.md:220]: Fixed all figure numbers in this file.

File: docs/documentation/maya/tools.md (hunk 1 above)
Comment (josevalverdeinbibo) [reply docs/documentation/maya/tools.md:131]: Done.

**Comment by lucasferryinbibo** [comment]
Please add a clarifying comment for Learn Muscle Patches @josevalverdeinbibo:

For the section where it explains that you can add a vertex mask please add something like (In tools page):

"If *Add Selection* is pressed with no selection it will display a pop-up and inform that all vertices will be used for the learning process. This button has to necessarily be pressed to enable the execution of the learning process."

**Review by carlosmonteagudoinbibo — APPROVED** [review]

**Pull request merged**